db/db_log_iter_test.cc

```diff
@@ -33,9 +33,8 @@ class DBTestXactLogIterator : public DBTestBase {
 };
 
 namespace {
-SequenceNumber ReadRecords(
-    std::unique_ptr<TransactionLogIterator>& iter,
-    int& count) {
+SequenceNumber ReadRecords(std::unique_ptr<TransactionLogIterator>& iter,
```

Comment:
Previously this always called `ASSERT_OK(iter->status())`, what changed that meant that sometimes this status needs to be checked as NOK?